Packs/WALLIXBastion/Integrations/WAB/WAB.py

```diff
@@ -11,34 +11,61 @@ class AuthError(Exception):
     pass
 
 
-def str_arg(args, name):
-    return str(args.get(name, ""))
+Unset = object()
 
 
```

Comment:
The functions were already here but they have been modified to handle cases were null values are expected and differ from absent values. Since this only apply to specific arguments of specific endpoints, I cannot easily use a general post-processing function, it was simpler to handle it in the argument parsers, with a distinctive value for null vs absent properties

Reply:
We think it is more elegant to use the original assign_params then convert the null value to None.
Would appreciate if you rethink about it.
Thank you!